lib/openstudio-standards/standards/ashrae_90_1/ashrae_90_1_2007/179d_ashrae_90_1_2007/179d_ashrae_90_1_2007.Model.rb

```diff
@@ -717,7 +717,11 @@ def model_create_prm_any_baseline_building(user_model, building_type, climate_zo
 
       # If there are any multi-zone systems, reset damper positions to achieve a 60% ventilation effectiveness minimum for the system
       # following the ventilation rate procedure from 62.1
-      model_apply_multizone_vav_outdoor_air_sizing(model)
+      if baseline_179d
+        OpenStudio.logFree(OpenStudio::Info, 'openstudio.standards.Model', '*** SKIP Baseline Multi-Zone VAV Outdoor Air Sizing ***')
+      else
+        model_apply_multizone_vav_outdoor_air_sizing(model)
```

Comment:
https://github.com/jmarrec/openstudio-standards/blob/8ceae8517e959485255f48424daba475b45459ad/lib/openstudio-standards/standards/Standards.Model.rb#L2243-L2255

https://github.com/jmarrec/openstudio-standards/blob/8ceae8517e959485255f48424daba475b45459ad/lib/openstudio-standards/standards/Standards.AirLoopHVAC.rb#L11-L22

in `def air_loop_hvac_adjust_minimum_vav_damper_positions(air_loop_hvac)`

https://github.com/jmarrec/openstudio-standards/blob/8ceae8517e959485255f48424daba475b45459ad/lib/openstudio-standards/standards/Standards.AirLoopHVAC.rb#L2193